homeassistant/components/minecraft_server/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Minecraft Server",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/minecraft_server",
-  "requirements": ["getmac==0.8.1", "mcstatus==2.3.0"],
+  "requirements": ["dnspython==1.16.0", "getmac==0.8.1", "mcstatus==2.3.0"],
```

Comment:
did you consider using aiodns ?

Reply:
In my research back then, I only found dnspython and another library supporting SRV records. But it looks like that aiodns also supports it. I'll give it a try and switch the dependency if possible. Thanks for the hint.